docs/source/usersguide/input.rst

```diff
@@ -1136,6 +1136,15 @@ Each ``material`` element can have the following attributes or sub-elements:
     .. note:: If one nuclide is specified in atom percent, all others must also
               be given in atom percent. The same applies for weight percentages.
 
+    An optional attribute/sub-element for each nuclide is ``scattering``. This
+    attribute may be set to "ace" to use the scattering laws specified in the
```

Comment:
How about "data" or "library" instead of "ace"? That way we won't have to change it in our post-ACE future.